README.md

```diff
@@ -39,44 +39,19 @@ See [Release procedure](CONTRIBUTING.md#release-procedure) for more information
 
 N/A
 
-## Install
+## Testnet
```

Comment:
Please rename this to something like "Temporary testnet setup instructions". These steps is not how it's going to work for the real testnet. This was setup that I wrote for our local docker setup which is not how it should work in real world.

Reply:
Agreed, although that reads to me as if the testnet is meant to be temporary. I have changed it to "Testnet Setup Instructions"